packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -146,18 +150,21 @@ SelectedFilters.propTypes = {
 	render: types.func,
 	title: types.title,
 	onChange: types.func,
+	componentProps: types.props,
 };
 
 SelectedFilters.defaultProps = {
 	className: null,
 	clearAllLabel: 'Clear All',
 	showClearAll: true,
 	style: {},
+	componentProps: {},
 };
 
 const mapStateToProps = state => ({
 	components: state.components,
 	selectedValues: state.selectedValues,
+	componentProps: state.props,
```

Comment:
@ShahAnuj2610 can you confirm once, if this component is re-rendering on any prop change in component. if so this needs to be optimized.

Reply:
sure